documentation/src/main/wiki/AdvancedSetup.md

```diff
@@ -0,0 +1,77 @@
+### Advanced Setup
+
+This section describes advanced setup of AET using Linux and Windows. 
+
+Please note that full list of required tools and its versions can be found in [System Components](SystemComponents) section.
+
+![aet-setup-advanced](assets/diagrams/aet-setup-advanced-grid-scaled.png)
+
+#### Linux Setup
+
+For Linux environment setup, use the [AET Cookbook](https://github.com/Cognifide/aet-cookbook)
```

Comment:
Let's list here, that cookbook setup provides:
- Karaf with AET service running
- MongoDB as the database
- ActiveMQ as the message broker
- BrowserMob as the proxy
- Selenium Grid Hub (that later Windows nodes will be attached to)
- Apache Serwer with the Report app

Reply:
Done